### DISCUSSION TITLE
can't use with mongoDB

### DISCUSSION THREAD

**Discussion by rishab0** [spatie/laravel-activitylog#788] [Q&A] [answered]
I got this error , when use this pakage
[http://i.imgur.com/IHsFgtJ.png](http://i.imgur.com/IHsFgtJ.png)

**Comment by Gummibeer** [comment, marked as answer]
Hey,
MongoDB isn't supported by eloquent or pdo. So you will have to go with a package like https://github.com/jenssegers/laravel-mongodb and create your own activity model extending the MongoDB base model and implementing the activity interface.
At the end MongoDB isn't in the list of officially supported databases as it's not by Laravel. You will be on your own doing it. The package should still work but isn't tested with MongoDB.
